app/components/inplace-boolean.js

```diff
@@ -1,14 +1,19 @@
 import Ember from 'ember';
-import InPlace from 'ilios/mixins/inplace';
+import InPlaceValidation from 'ilios/mixins/inplace-validation';
```

Comment:
If it is just so everything is using the same mixin then you should remove the plain inplace and just keep the inplace-validation.